### ISSUE TITLE
ansible-runner doesn't support multiple inventories in the way that all other ansible commands do

### ISSUE THREAD

**Issue by willthames** [ansible/ansible-runner#296]
ansible-runner doesn't seem to cope with ANSIBLE_INVENTORY or inventory being set in ansible.cfg

While the `--inventory` flag is nice, it relies on the calling code having to parse inventory arguments rather than just inheriting it from the environment

In an ideal world, if `ansible-inventory --list` works with no additional arguments, then so should `ansible-runner -p playbook.yml run /working_dir`

**Comment by willthames** [comment]
I say multiple inventories in my title because `ansible-inventory --list` works when 

```
ANSIBLE_INVENTORY=/opt/ansible/inventory,/applications/docker-debug/docker-debug/inventory,/applications/kuard/kuard/inventory
```

whereas I'd have to use `--inventory /opt/ansible/inventory --inventory /applications/docker-debug/docker-debug/inventory --inventory /applications/kuard/kuard/inventory`

**Comment by matburt** [comment]
This is similar to https://github.com/ansible/ansible-runner/issues/245 and I think the solution I laid out works will work for this here also.

**Comment by matburt** [comment]
Fixed via #309
